.github/workflows/test.yml

```diff
@@ -25,20 +25,20 @@ jobs:
       - name: set build cache
         uses: actions/cache@v3
         with:
-            path: |
-                ~/.cargo/bin/
-                ~/.cargo/registry/index/
-                ~/.cargo/registry/cache/
-                ~/.cargo/git/db/
-                digital-asset-rpc-infrastructure/target/
-            key: cargo-${{ hashFiles('**/Cargo.lock') }}-0001
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            digital-asset-rpc-infrastructure/target/
+          key: cargo-${{ hashFiles('**/Cargo.lock') }}-0001
 
       # Cargo.lock
       - name: Check lock file
         run: |
           cargo tree
           git checkout Cargo.lock
-          cargo tree --frozen
```

Comment:
yellowstone crates are referenced using git so require network fetch to retrieve which violates frozen flag.

Reply:
https://lib.rs/crates/yellowstone-grpc-client

NVM crate is published